src/eradiate/thermoprops/afgl_1986.py

```diff
@@ -8,10 +8,24 @@
 import pint
 import xarray as xr
 
-import eradiate
-
 from .util import compute_scaling_factors, interpolate, rescale_concentration
 from .. import data
+from .._mode import mode
+
+
+def validate_model_id(value: str):
+    valid_values = [
+        "tropical",
+        "midlatitude_summer",
+        "midlatitude_winter",
+        "subarctic_summer",
+        "subarctic_winter",
+        "us_standard",
+    ]
+    if not value in valid_values:
+        raise ValueError(
+            f"model_id should be in {','.join(valid_values)} (got {value})"
+        )
```

Comment:
If I'm not mistaken, this function is called only once. I don't really see the point of keeping it; instead, I'd suggest to just keep the list of valid IDs as a constant:
```suggestion
VALID_MODEL_IDS = [
    "tropical",
    "midlatitude_summer",
    "midlatitude_winter",
    "subarctic_summer",
    "subarctic_winter",
    "us_standard",
]
```
and later, explicitly write those two lines. An added benefit is that you can reuse this list later (*e.g.* in your tests).